actions/wait-for-deployment/README.md

```diff
@@ -19,4 +24,13 @@ Ensure that the UI pod is running in the namespace before continuing
 NAMESPACEINUSE:
   description: Wait for the UI pod to run in this namespace before continuing
   default: none
+DEPLOYMENT_TYPE
+  description: Declare 'api' or 'ui' deployment
+  default: none
+API_URL
+  description: The url to check the /configuraiton endpoint for the current running version
+  default: none
+DESIRED_GITHASH
+  description: The git hash against which to compare the current running version
+  default: none
```

Comment:
The method for determining these variables varies pretty heavily between the perf and integration test. I thought it was better to have that logic live in the different workflows rather than adding more "if else" statements in this action, but can be convinced otherwise!

For example:
"DESIRED_GITHASH":
integration - determined by the current branch - git rev-parse --short HEAD
perf - retrieved from dev's configuration endpoint's "gitHash" value so that we're running against the latest version

Reply:
Yes, this in this case it seems unavoidable.